api/v2/guides/integration-javascript-server-side.md

```diff
@@ -0,0 +1,196 @@
+
+The following sections provide additional details for each step in the diagram:
+ 
+ 1. [Establish identity: User Login](#establish-identity-user-login)
+ 2. [Bid Using EUID Tokens](#bid-using-euid-tokens)
+ 3. [Refresh Tokens](#refresh-tokens)
+ 4. [Clear Identity: User Logout](#clear-identity-user-logout)
+
+### Establish Identity: User Login
+
+After authentication in step 1-c, which forces the user to accept the rules of engagement and allows the publisher to validate the user's email address, an EUID token must be generated on the server side. The following table details the token generation steps.
+
+| Step | Endpoint/SDK | Description |
+| :--- | :--- | :--- |
+| 1-d | [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) | After the user authenticates and authorizes the creation of an EUID, use the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint to generate an EUID token using the email address provided by the user. Make sure it is normalized. |
+| 1-e | [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) | The endpoint returns an EUID token generated from the user's hashed or unhashed email address. |
+| 1-f | EUID SDK for JavaScript | The SDK sends the returned EUID token from step 1-e to the SDK in the `identity` property of its [init() function](../sdks/client-side-identity.md#initopts-object-void). |
+| 1-g | EUID SDK for JavaScript | Provide the SDK a callback function that will receive identity updates from the SDK and use them to initiate targeted advertising. |
+
+#### Example: JavaScript
+
+```js
+  window.__uid2 = window.__uid2 || {};
```

Comment:
All instances of `__uid2` in this doc should instead be `__euid`. @genwhittTTD do you want one of the devs to do it, or are you comfortable with that?

Reply:
The same thing applies to api/v2/guides/publisher-client-side.md.